### PR TITLE
Obligatory screenshot description

### PR DESCRIPTION
**Pull request by johanmalm**

### CODE REVIEW

**Comment by ludg1e** [comment]
LGTM apart from the comments

**Comment by johanmalm** [comment]
Comments? Have you clicked 'complete review'

**Comment by ludg1e** [comment]
> Comments? Have you clicked 'complete review'

I mean, I've started a review, and added on the review some comments

File: src/integration.md [hunk 1]
```diff
@@ -86,6 +86,17 @@ Configure sfwbar in the `~/.config/sfwbar/sfwbar.config` file.
 The default config will mostly work however for more information see the
 [sfwbar man page].
 
+[sfwbar/config] contains example config files (with file extension `.config`)
+which can be by merely copying them to `~/.config/sfwbar/sfwbar.config`. Note
```
Comment (ludg1e) [inline src/integration.md:90]: Shouldn't it be "which can be used by merely" ... ?
English isn't my native language though

Maybe “applied” fits better

File: src/obligatory-screenshot.md [hunk 2]
```diff
@@ -0,0 +1,198 @@
...
+1. [Introduction](#introduction)
+2. [Background](#background)
+3. [Tweaks](#tweaks)
+4. [Theme](#theme)
+5. [Panel](#panel)
+6. [Screenshots](#screenshots)
+
+# 1. Introduction {#introduction}
+
+The [obligatory screenshot] used on the `labwc` website index page and github
```
Comment (ludg1e) [inline src/obligatory-screenshot.md:12]: GitHub (super optional)

File: src/integration.md (hunk 1 above)
Comment (johanmalm) [reply src/integration.md]: Thanks. Done

File: src/obligatory-screenshot.md (hunk 2 above)
Comment (johanmalm) [reply src/obligatory-screenshot.md]: Done

**Comment by Consolatis** [comment]
> > Comments? Have you clicked 'complete review'
> 
> I mean, I've started a review, and added on the review some comments Oh gosh
> 
> Edit: done I think

This is something that I bet every single GH user doing reviews managed to do in the past. I know for sure that I did a review, wondered why nobody responded and then some days later figured out that the review was still stuck on my end.

**Pull request merged**